vic/drivers/classic/src/vic_classic.c

```diff
@@ -109,17 +109,17 @@ main(int   argc,
     // Check that model parameters are valid
     validate_parameters();
 
+    /** Make Date Data Structure **/
+    initialize_time();
+    dmy = make_dmy(&global_param);
+
     /** Set up output data structures **/
-    out_data = create_output_list();
+    set_output_met_data_info();
+    out_data = create_outdata(1);
```

Comment:
add a comment here so its clear that we're 1 is because the classic driver operates on 1 grid cell at a time.